core/src/test/scala/org/broadinstitute/dsde/rawls/webservice/EntityApiServiceSpec.scala

```diff
@@ -3068,31 +3068,30 @@ class EntityApiServiceSpec extends ApiServiceSpec {
   }
 
   // TODO CORE-635 Update quicksilver to have same behavior as legacy, then update test to use quicksilver
-  it should "return 200 OK on entity query for unknown sort field" in withLegacyPaginationTestDataApiServices {
-    services =>
-      Get(
-        s"${legacyPaginationTestData.workspace.path}/entityQuery/${legacyPaginationTestData.entityType}?sortField=asdfasdfasdf"
-      ) ~>
-        sealRoute(services.entityRoutes(userInfo = userInfo)) ~>
-        check {
-          assertResult(StatusCodes.OK) {
-            status
-          }
-          assertResult(
-            EntityQueryResponse(
-              defaultQuery.copy(sortField = "asdfasdfasdf"),
-              EntityQueryResultMetadata(
-                legacyPaginationTestData.numEntities,
-                legacyPaginationTestData.numEntities,
-                calculateNumPages(legacyPaginationTestData.numEntities, defaultQuery.pageSize)
-              ),
-              legacyPaginationTestData.entities.sortBy(_.name).take(defaultQuery.pageSize)
-            )
-          ) {
+  it should "return 200 OK on entity query for unknown sort field" in withPaginationTestDataApiServices { services =>
```

Comment:
once you convert all the pagination tests, you can get rid of `withLegacyPaginationTestDataApiServices`

Reply:
done in [57d6fb6](https://github.com/broadinstitute/rawls/pull/3443/commits/57d6fb6b817103e00069a29135687ee07f4a45f5)